.azure/pipelines/ci.yml

```diff
@@ -93,17 +93,17 @@ variables:
   - name: _InternalRuntimeDownloadCodeSignArgs
     value: ''
 - ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-  - group: DotNet-MSRC-Storage
+  - group: DotNetBuilds storage account read tokens
   - name: _InternalRuntimeDownloadArgs
-    value: -RuntimeSourceFeed https://dotnetclimsrc.blob.core.windows.net/dotnet
-           -RuntimeSourceFeedKey $(dotnetclimsrc-read-sas-token-base64)
-           /p:DotNetAssetRootAccessTokenSuffix='$(dotnetclimsrc-read-sas-token-base64)'
+    value: -RuntimeSourceFeed https://dotnetbuilds.blob.core.windows.net/internal
+           -RuntimeSourceFeedKey $(dotnetbuilds-internal-container-read-token-base64)
+           /p:DotNetAssetRootAccessTokenSuffix='$(dotnetbuilds-internal-container-read-token-base64)'
```

Comment:
Not sure about this. Don't we want to put all of the conditions here or nearby❔ That is, choose one of the property names and provide different values here, depending on the environment / situation. It gets confusing though. `/internal` should only be used in specific cases, not for all internal builds☹️

Reply:
Don't we always want this to be dotnetbuilds/internal? This is the backup location, so if we can't download the runtime from the primary (public) feed, this is always the one we fall back one - is that right @dotnet/dnceng @mmitche?

Reply:
/cc @alexperovich might have some context too?

But dotnetbuilds/internal is the direct replacement for daily builds for dotnetmsrc, as far as I know.

Reply:
Yes, that's correct. Use of dotnetclimsrc is gone in 6.0/main

Reply:
Yep, dotnetbuilds/internal is a complete replacement for dotnetclimsrc.